source/libs/api.ts

```diff
@@ -26,21 +26,26 @@ so the call will not throw an error but it will return as usual.
 
 import OptionsSync from 'webext-options-sync';
 
-export const v3 = (...args) => call(fetch3, ...args);
-export const v4 = (...args) => call(fetch4, ...args);
-export const escapeKey = string => '_' + string.replace(/[./-]/g, '_');
+type FetchStrategy = typeof fetch3 | typeof fetch4;
+type CallArguments = TailArgs<ArgumentTypes<typeof call>>;
+
+export const v3 = (...args: CallArguments) => call(fetch3, ...args);
```

Comment:
I feel like the use of an `args` parameter here is completely unnecessary TBH. Even the comment in the file header describes the use of the API with only 2 arguments, so why was decided to take an `args` parameter in the first place?

From the point of view of a consumer of either v3 or v4, there is practically no information about the arguments that any of those functions expect, despite the fact that the header comment is really clear.

See https://github.com/jotatoledo/refined-github/blob/ef0883e8dad72541da6cdd0eabd21b627ba71eab/source/libs/api.ts#L29 for an alternative approach.

Reply:
I typed `...args` as I was trying to not change the code too much, just the types, but agreed, it's not necessary. I'll make the change tonight.

Reply:
done.